cloud/multi-factor-authentication.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Multi-factor user authentication
+excerpt: Manage Multi-factor user authentication for your Timescale Cloud account
+product: cloud
+keywords: [mfa, accounts, admin]
+tags: [two-factor user authentication]
+---
+
+## Multi-factor user authentication
+
+You can use two-factor authentication also known as 2-Step verification or 2FA
+to log in to your Timescale Cloud account. This enables secure logins that
```

Comment:
```suggestion
You can use two-factor authentication to log in to your Timescale Cloud 
account. Two-factor authentication, also known as two-step verification 
or 2FA, enables secure logins that
```

Just trying to make sure that the first sentence doesn't get derailed too badly, and try and keep the focus on things you can do with Cloud, rather than what on earth is 2FA.